sources/packages/backend/libs/sims-db/src/database.module.ts

```diff
@@ -7,7 +7,7 @@ import { DBEntities, ormConfig } from "./data-source";
   imports: [
     TypeOrmModule.forRoot({
       ...ormConfig,
-      logging: ["error", "warn"],
+      logging: ["error", "warn", "query"],
```

Comment:
Is it intentional?

Reply:
Sorry, got pushed by mistake. Will remove it.